docs: update quick-start-guide for admin UIs and Perl debugger (#12808)
openfoodfacts/openfoodfacts-server: docs/dev/how-to-quick-start-guide.md

* `COMPOSE_FILE` is the `;`-separated list of Docker compose files that are included in the deployment:
  * For a **development**-like environment, set it to `docker-compose.yml;docker/dev.yml` (default)
  * For a **production**-like environment, set it to `docker-compose.yml;docker/prod.yml;docker/mongodb.yml`

>   *For more features, you can add containers from `shared-services`:

Reviewer: ```suggestion
  * For more features, you can add containers from `shared-services`:
```

 * **Admin UIs**: now available in `shared-services/admin-uis.yml` (the old `docker/admin-uis.yml` is no longer supported)
  * **Geolite2**: continue using `docker/geolite2.yml`

>   * **Perl debugger**: this feature is currently not available (`docker/perldb.yml` is outdated)

Reviewer: This is a “quick start guide”. The primary audience are people who are setting up product opener for the first time. “continue using” doesn’t make sense, as they’re not using anything currently, so they have nothing to continue using. Also, as I understand it, they are optional add-ons that you can choose or not choose to include.

There’s also no need to mention old locations (or things that are not currently available). The reader can be presumed to have no prior context/knowledge, and this is just extra information they have to consider. This also refers to temporal statements like “now available”. Just “available” works fine.

> 

Reviewer: Indentation is both inconsistent within this sub-list, and also out of alignment with the “parent” list.

> `COMPOSE_SEPARATOR` defines the separator used for the `COMPOSE_FILE`.

Reviewer: Did you intentionally take this out of the list?
```suggestion
* `COMPOSE_SEPARATOR` defines the separator used for the `COMPOSE_FILE`.
```

>   *For more features, you can add containers from `shared-services`:
> 
>  * **Admin UIs**: now available in `shared-services/admin-uis.yml` (the old `docker/admin-uis.yml` is no longer supported)
>   * **Geolite2**: continue using `docker/geolite2.yml`
>   * **Perl debugger**: this feature is currently not available (`docker/perldb.yml` is outdated)
> 
> `COMPOSE_SEPARATOR` defines the separator used for the `COMPOSE_FILE`.

Reviewer: A much less invasive change would just be updating the path and removing the no-longer relevant entry:
```suggestion
  * For more features, you can add:
    * `shared-services/admin-uis.yml`: add the Admin UIS container
    * `docker/geolite2.yml`: add the Geolite2 container
* `COMPOSE_SEPARATOR` is the separator used for `COMPOSE_FILE`.
```

It does seem like you’re doing some additional formatting and rewording changes, so I’ll leave it to others’ judgement whether these formatting and rewording changes improve things.
